GenderPayGap.WebUI/wwwroot/styles/components/_number-icons.scss

```diff
@@ -0,0 +1,37 @@
+﻿.action-steps {
+  position: relative;
+  padding-bottom: 10px;
+  padding-left: -30px;
+
+  ol {
+    list-style: none;
+    counter-reset: action-step-counter;
+    padding-left: 0;
+  }
+  ol > li {
+    counter-increment: action-step-counter;
+
+    @media (max-width: map-get($govuk-breakpoints, 'desktop')) {
+      > h3 {
+        margin-left: 30px;
+      }
+    }
+  }
+  ol > li::before {
+    content: counter(action-step-counter);
+    position: absolute;
+    line-height: 23px;
+    font-size: small;
+    text-align: center;
+    vertical-align: middle;
+    width: 23px;
+    height: 23px;
+    color: white;
+    background: black;

```

Comment:
If you wanted to be _REALLY_ nitpicky, these should be `$govuk-text-colour` and `$govuk-background-colour` - since they are the versions of black and white that are defined elsewhere - and indeed because it's _not_ #000000, it's #0b0c0c.